scripts/zones/Norg/npcs/_700.lua

```diff
@@ -94,9 +94,10 @@ function onEventFinish(player,csid,option)
         player:completeMission(ROV, tpz.mission.id.rov.WHAT_LIES_BEYOND)
         player:addMission(ROV, tpz.mission.id.rov.THE_TIES_THAT_BIND)
     elseif csid == 279 then
-        player:addItem(10159) -- Cipher: Zeid II
-        player:completeMission(ROV, tpz.mission.id.rov.VOLTO_OSCURO)
-        player:addMission(ROV, tpz.mission.id.rov.RING_MY_BELL)
+        if npcUtil.giveItem(player, 10160) then -- Cipher: Zeid II
+            player:completeMission(ROV, tpz.mission.id.rov.VOLTO_OSCURO)
+            player:addMission(ROV, tpz.mission.id.rov.RING_MY_BELL)
+        end
```

Comment:
Could we change this to:
```lua
        if npcUtil.giveItem(player, 10160) then -- Cipher: Zeid II
            player:completeMission(ROV, tpz.mission.id.rov.VOLTO_OSCURO)
            player:addMission(ROV, tpz.mission.id.rov.RING_MY_BELL)
        else
            player:setCharVar("ZeidIICipher", 1)
        end
```
and then above change to:
```lua
    elseif player:getCharVar("ZeidIICipher") == 1 then
        if npcUtil.giveItem(player, 10160) then -- Cipher: Zeid II
            player:completeMission(ROV, tpz.mission.id.rov.VOLTO_OSCURO)
            player:addMission(ROV, tpz.mission.id.rov.RING_MY_BELL)
        end
    elseif RhapsodiesMission == tpz.mission.id.rov.VOLTO_OSCURO then
        player:startEvent(279)
```
apparently RoV doesn't make you rewatch the CS.

Reply:
RoV introduced those sparkle NPCs that store the item you should have gotten - mystic retriever.

Reply:
Mystic retrievers are used most of the time, but Gilamesh's door is special and hands it to you directly.

Source: "Rhapsodies: ROV 1-17 ~ ROV 2-2 (Already Meeting COP Reqs)" @ 3:30

Reply:
resolving this per @ibm2431 's comments, no changes made.

Reply:
It's still not supposed to make you watch the CS again.